src/mongodb/scores.controller.ts

```diff
@@ -4452,10 +4454,10 @@ export class ScoresController {
   @Post('/getUsersFamalarity')
```

Comment:
@DevendraPPatil 
Can you please correct familiarity spellings here for endpoint and also for function